### PR TITLE
Couple of fixes for Quarto

### PR DESCRIPTION
**Pull request by yongrenjie**
For Quarto x Plotly issue, see: https://github.com/quarto-dev/quarto-cli/discussions/5264 for the solution

### CODE REVIEW

File: stories/2024-01-01-Eurovision/story.ipynb [hunk 1]
```diff
@@ -1238,6 +1238,10 @@
     "#| code-fold: true\n",
     "import plotly.express as px\n",
     "\n",
+    "# The following lines are needed for Quarto to render the interactive plot\n",
+    "import plotly.io as pio\n",
+    "pio.renderers.default = \"notebook\"\n",
```
Comment (llewelld) [inline stories/2024-01-01-Eurovision/story.ipynb]:
```suggestion
    "pio.renderers.default = \"plotly_mimetype+notebook_connected\"\n",
```
With this change I find it shows beautifully in Qarto, but then no longer shows in Jupyter. With the above (taken from [the docs](https://quarto.org/docs/interactive/widgets/jupyter.html#plotly)) I get it working in both. Does this affect VS Code though? Or maybe this is just a glitch on my system?

**Review by llewelld — COMMENTED** [review]
I have a query as this now works with Quarto but not with Jupyter Lab. I think working with Quarto is better if it's one or the other, but I'm wondering if it's possible to get it working on everything.

File: stories/2024-01-01-Eurovision/story.ipynb (hunk 1 above)
Comment (yongrenjie) [reply stories/2024-01-01-Eurovision/story.ipynb]: Works on VSCode and Quarto

**Review by llewelld — APPROVED** [review]
Great, LGTM, thank you!

**Comment by yongrenjie** [comment]
Will redo changes and commit directly to `eurovision-main` due to merge conflicts

**Pull request closed without merging**